File: app/src/main/java/com/dpis/module/home/ModeGuideActivity.java
package com.dpis.module.home;

import com.dpis.module.settings.LocalizedActivity;
import com.dpis.module.about.presentation.SupportActivityContent;

import android.os.Bundle;

public final class ModeGuideActivity extends LocalizedActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        SupportActivityContent.installModeGuide(this);
    }
}

File: app/src/main/java/com/dpis/module/home/ModeGuideActivity.kt
package com.dpis.module.home

import android.os.Bundle
import com.dpis.module.home.presentation.installModeGuide
import com.dpis.module.settings.LocalizedActivity

class ModeGuideActivity : LocalizedActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        installModeGuide()
    }
}
